.devcontainer/core-dev/devcontainer.json

```diff
@@ -82,6 +82,8 @@
     "-v", "${localWorkspaceFolder}/python/Gemfile:/home/dependabot/dependabot-core/python/Gemfile",
     "-v", "${localWorkspaceFolder}/python/lib:/home/dependabot/dependabot-core/python/lib",
     "-v", "${localWorkspaceFolder}/python/spec:/home/dependabot/dependabot-core/python/spec",
+    "-v", "${localWorkspaceFolder}/swift/lib:/home/dependabot/dependabot-core/swift/lib",
+    "-v", "${localWorkspaceFolder}/swift/spec:/home/dependabot/dependabot-core/swift/spec",
```

Comment:
I see that other ecosystems also add the `.gemspec` and `Gemfile` files, should we add them here too, or remove the others if not needed?

Reply:
I think @Nishnha published a Gem for Swift; should we add it for consistency?

Reply:
Done

Reply:
yeah we should also mount those